package.json

```diff
@@ -33,6 +33,7 @@
     "material-ui-dropzone": "^3.0.0",
     "moment": "^2.24.0",
     "react": "^16.12.0",
+    "react-beautiful-dnd": "^13.0.0",
```

Comment:
Remove old library

Reply:
This is the new one, but good point, I'll remove the old nevertheless.